src/sdk/services/oauthService.ts

```diff
@@ -24,11 +22,28 @@ class OAuthService {
     return originUrl.origin.replace(regex[1], 'accounts');
   };
 
-  private getOAuthAuthorizeUrl = (api: ManifestApi) => {
+  private getOAuthAuthorizeUrl = (api: ManifestApi, redirectUri?: string) => {
     const host = this.getOAuthHost();
-    const scopes = encodeURIComponent(api.scopes.join(' '));
-    const redirectUri = encodeURIComponent(api.redirectUri);
-    return `${host}/oauth/authorize?client_id=${api.applicationId}&redirect_uri=${redirectUri}&response_type=code&scope=${scopes}`;
+    const scopes = encodeURIComponent(api.scopes.join(','));
+    const selectedRedirectUri = encodeURIComponent(this.selectRedirectUri(api, redirectUri));
+    const clientId = api.client.id ?? api.applicationId;
```

Comment:
Should be encoded as well.
```suggestion
    const clientId =  encodeURIComponent(api.client.id ?? api.applicationId);
```

Reply:
Fixed

Reply:
I don't see the fix yet

Reply:
You won't see it in the conversation, you have to go though the PR files again. Github UX is not ideal 👎